crates/astria-conductor/src/executor.rs

```diff
@@ -226,21 +279,53 @@ impl<C: ExecutionClientV1Alpha1> Executor<C> {
 
         let response = self
             .execution_rpc_client
-            .call_do_block(prev_block_hash, block.rollup_transactions, Some(timestamp))
+            .call_execute_block(prev_block_hash, block.rollup_transactions, Some(timestamp))
             .await?;
-        self.execution_state = response.block_hash.clone();
+        self.execution_state = response.hash.clone();
 
         // store block hash returned by execution client, as we need it to finalize the block later
         info!(
             sequencer_block_hash = ?block.block_hash,
             sequencer_block_height = block.header.height.value(),
-            execution_block_hash = hex::encode(&response.block_hash),
+            execution_block_hash = hex::encode(&response.hash),
             "executed sequencer block",
         );
         self.sequencer_hash_to_execution_hash
-            .insert(block.block_hash, response.block_hash.clone());
+            .insert(block.block_hash, response.hash.clone());
 
-        Ok(Some(response.block_hash))
+        Ok(Some(response.hash))
+    }
+
+    /// Updates the block's commitment state to firm
+    ///
+    /// # Arguments
+    ///
+    /// * `block` - the sequencer block that we know is firm
+    /// * `execution_block_hash` - the execution block hash corresponding to the sequencer block
+    async fn update_commitment_state_firm(
```

Comment:
I think we should make this just `update_commitment_state` and have it take both the soft and firm commitment blocks as args?

There's no API to update one without the other anymore.